simpleclient_pushgateway/src/main/java/io/prometheus/client/exporter/PushGateway.java

```diff
@@ -54,15 +56,42 @@
  */
 public class PushGateway {
 
-  private final String address;
+  // Visible for testing
```

Comment:
Do you think `protected` is better? Package local is the strictest thing available (protected allows access from both package and inheritance), but I'll change it if you prefer the added access from heirs.